src/lTerm_widget.mli

```diff
@@ -33,6 +33,13 @@ class t : string -> object
   method can_focus : bool
     (** Whether the widget can receive the focus or not. *)
 
+  method focus : (t option * t option * t option * t option) option
```

Comment:
I did actually implement it with a record to begin with, however, I couldn't find the proper place to put it.  I ended up with a new module `LTerm_focus` with just this type which seemed overkill.  Maybe it could live in LTerm_geom though?

With this change shall we also change `method set_focus : t option directions -> unit`?

Will remove the extra option.